astropy/coordinates/tests/test_solar_system.py

```diff
@@ -55,13 +55,13 @@ def test_positions_skyfield(tmpdir):
     # skyfield ephemeris
     try:
         planets = load('de421.bsp')
+        ts = load.timescale()
```

Comment:
It might go against the intention here, but `load.timescale(builtin=True)` will suppress file download and use built-in copies of the leap second files instead.

Reply:
I thought about that but I don't know what is really being tested here and the test is already marked as a remote data test. Maybe @adrn or @eteq can clarify.

Reply:
This was there already, just moved up so that if, somehow, `skyfield` tries to access the internet, we'll fail on jobs where we do not allow that. I think it is OK.

Also: the tests here are for specific times in the past, so should be OK not to update the leap-second list!

Reply:
@mhvk , did you mean that I should have used `ts = load.timescale(builtin=True)` and not try to ignore its timeout error in CI? Can you please clarify? If so, I can open a follow-up PR.

Reply:
No, sorry, I misunderstood here - this remains a remote-date test and what you did is exactly the right thing!